### PR TITLE
Update large_model_inference.md

### PR DESCRIPTION
**Pull request by sekyondaMeta**
## Description

Adding DeepSpeed MII and Hugging face accelerate information to LMI doc.

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] New feature (non-breaking change which adds functionality)
- [x] This change requires a documentation update

## Feature/Issue validation/testing

Built pages locally.

## Checklist:

- [x] Did you have fun?
- [ ] Have you added tests that prove your fix is effective or that this feature works?
- [ ] Has code been commented, particularly in hard-to-understand areas?
- [x] Have you made corresponding changes to the documentation?

### CODE REVIEW

**Review by msaroufim — APPROVED** [review]

File: docs/large_model_inference.md [hunk 1]
```diff
@@ -1,6 +1,13 @@
 # Serving large models with Torchserve
 
 This document explain how Torchserve supports large model serving, here large model refers to the models that are not able to fit into one gpu so they need be split in multiple partitions over multiple gpus.
+This page is split into the following sections:
+- [How it works](#how-it-works)
```
Comment (msaroufim) [inline docs/large_model_inference.md:5]: I remember there was some issue with relative links when we go to pytorch.org so might wanna double check this works, if it does feel free to dismiss this comment

File: docs/large_model_inference.md [hunk 2]
```diff
@@ -175,7 +182,20 @@ torch-model-archiver --model-name bloom --version 1.0 --handler deepspeed_handle
...
-## Best Practices
+## DeepSpeed MII
+If working with one of the supported models shown [here](https://github.com/microsoft/deepspeed-mii#supported-models-and-tasks) you can take advantage of Deep Speed MII. Deep Speed MII uses Deep Speed Inference along with further advances in deep learning to minimize latency and maximize throughput. It does this for specific model types, model sizes, batch sizes and available hardware resources.
+
+For more information on how to take advantage of Deep Speed MII on supported models, see the information [here](https://github.com/microsoft/DeepSpeed-MII).
+You can also find an example of how to apply this to TorchServe [here](https://github.com/pytorch/serve/tree/master/examples/large_models/deepspeed_mii).
+
+## Serving Large Hugging Face Models Using Accelerate
+
+If working with large Hugging Face models but have limited resources, you can use accelerate to serve these models. To achieve this, you would need to set `low_cpu_mem_usage=True` in the config.json file.
```
Comment (agunapal) [inline docs/large_model_inference.md:193]: @HamidShojanazeri  Is this config still valid?

**Review by agunapal — COMMENTED** [review]
Pending one query about `low_cpu_mem_usage=True`, rest looks fine

File: docs/large_model_inference.md (hunk 2 above)
Comment (HamidShojanazeri) [reply docs/large_model_inference.md:193]:
```suggestion
If working with large Hugging Face models but have limited resources, you can use accelerate to serve these models. To achieve this, you would need to set `low_cpu_mem_usage=True` and set the `device_map="auto" in the setup_config.json file.
```

**Review by HamidShojanazeri — APPROVED** [review]
Left a suggestion

**Pull request merged**